modules/config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     return typeof _matchMediaFunction === 'function';
   },
 
-  matchMedia (query: string) {
+  matchMedia (query: string): Object {
```

Comment:
Should be an object with addListener and removeListener functions: https://developer.mozilla.org/en-US/docs/Web/API/MediaQueryList

The more specific we get, the more bugs flow can catch :yellow_heart: